js/utils/musicutils.js

```diff
@@ -373,6 +373,79 @@ var OSCTYPES = [
     [_('sawtooth'), 'sawtooth'],
 ];
 
+const TEMPERAMENT = {
+    'equal': {
+        'ratios': {
```

Comment:
Do we need to define the intervals inside of a ratios object? Why the extra layer?
e.g., TEMPERAMENT['equal']['ratios']['unison'] could just be TEMPERAMENT['equal']['unison'].